src/ui/common/src/components/integrations/cards/ecrCard.tsx

```diff
@@ -0,0 +1,39 @@
+import Box from '@mui/material/Box';
+import React from 'react';
+
+import { ECRConfig, Integration } from '../../../utils/integrations';
+import { ResourceCardText } from './text';
+
+type ECRCardProps = {
+  integration: Integration;
+};
+
+export const ECRCard: React.FC<ECRCardProps> = ({ integration }) => {
+  const config = integration.config as ECRConfig;
+
+  const labels = [];
+  const values = [];
+
+  if (config.region) {
+    labels.push('Region');
+    values.push(config.region);
+  }
+
+  if (config.config_file_path) {
+    labels.push('Credential File Path');
+    values.push(config.config_file_path);
+  }
+
+  if (config.config_file_profile) {
+    labels.push('Profile');
+    values.push(config.config_file_profile);
+  }
+
+  return (
+    <Box>
```

Comment:
This box isn't really needed anymore. you can just return the ResourceCardText component.